### PR TITLE
[DOC] Enhance documentation

### PR DESCRIPTION
**Pull request by GradedJestRisk**
Enhance REAME.md by:
- fixing typos;
- cleaning up markdown tags;
- adding structure;
- mentions install requirements.

My 2 cents !
PR is split up in several commits, so that when a suggestions is obviously wrong, it can be easily discarded.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,33 @@
 # EasterEig
 ------------
 
-Consider a parametric eigenvalue problem depending on a parameter \(\nu\). This arises for instance in
+Consider a parametric [eigenvalue](https://en.wikipedia.org/wiki/Eigenvalues_and_eigenvectors) problem depending on a parameter \(\nu\). This arises for instance in
```
Comment (nennigb) [inline README.md:4]: gh
```suggestion
Consider a parametric problem depending on a parameter \(\nu\). This arises for instance in
```

Comment (GradedJestRisk) [reply README.md:4]: This was superfluous

File: README.md [hunk 2]
```diff
@@ -58,16 +58,28 @@ Tested for python 3.5
 > To run an example with petsc (parallel), you need to run python with `mpirun`. For instance, to run a program with 2 proc
 > `mpirun -n 2 python3 myprog.py`
 
-## Install (without pip)
+Install 
 --------------------------------------
+You'll need:
+* python; 
+* python3-tk library;
```
Comment (nennigb) [inline README.md:65]: not sure about ` python3-tk` dependency. Could you remove it? and just say that python installation are required.
Ok for all other modifications.

**Comment by GradedJestRisk** [comment]
As you already made the fix by yourself, I'll close this PR

**Pull request closed without merging**